ipa-core/src/protocol/ipa_prf/aggregation/step.rs

```diff
@@ -9,28 +9,19 @@ pub(crate) enum AggregationStep {
 }
 
 #[derive(CompactStep)]
-pub enum BucketStep {
-    /// should be equal to `MAX_BREAKDOWNS`
-    #[step(count = 512, child = crate::protocol::boolean::step::BoolAndStep)]
-    Bit(usize),
-}
-
-impl From<u32> for BucketStep {
-    fn from(v: u32) -> Self {
-        Self::Bit(usize::try_from(v).unwrap())
-    }
-}
+#[step(count = 512, child = crate::protocol::boolean::step::EightBitStep, name = "b")]
```

Comment:
I think we should keep the comment about keeping this in sync with MAX_BREAKDOWNS.